pages/wiki/docserver.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Local containerized documentation server
+layout: documentation
+---
+
+This web site can be built and run within a software container, such as those used *container engines* such as [Docker](https://www.docker.com/) or [podman](https://podman.io/). One reason for doing this is to be able to install and build the documentation without having to load the collection of tools on your real computer. Building and running a documentation server in a local container is the topic of this page.
+
+# Building the containerized version
+---
+
+This documentation will use `podman`, but the commands for `docker` are identical. If you wish to use `docker` instead of `podman`, just substitute `docker` for `podman` in each of the commands shown below. Any differences are explicitly noted.
+
+``` Bash
+podman build https://raw.githubusercontent.com/AsteroidOS/asteroidos.org/master/Dockerfile -t asteroidos-docserver
+```
+
+This will create an `asteroidos-docserver` image which is only around 236 MB and based on the Alpine Linux distribution. To run it locally:
+
+``` Bash
+podman run -p8080:80 asteroidos-docserver
+```
+
+This will run a server in the container which can then be reached on your local machine by pointing your browser to `localhost:8080/`.
```

Comment:
So I have a tendency to think that if we document the same thing in two different places (both the asteroidos.org repo's README and the project doc) we will sooner or later have the two set of instructions diverge. wdyt ?

Reply:
I agree.  Maybe it should just be on the web site and the README can point to it?

Reply:
Sgtm

Reply:
Should I then also remove the original build instructions from the README.md and just point to the web site?  Seems like that might be cleaner and clearer.

Reply:
yep